frontend/public/reducers/features.ts

```diff
@@ -52,7 +52,15 @@ plugins.registry
     }
   });
 
-const pluginFlagNames = _.uniq(plugins.registry.getFeatureFlags().map((ff) => ff.properties.flag));
+const pluginFlagNames = _.uniq(
```

Comment:
Note, there is also

```ts
CRDs[modelRef] = ff.properties.flag as FLAGS;
```

which force-casts the property we are changing in this PR.